config/default/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 images:
 - name: kube-rbac-proxy
   newName: gcr.io/kubebuilder/kube-rbac-proxy
-  newTag: v0.5.0
+  newTag: v0.8.0
```

Comment:
Sorry to nit: is this intended? Related to prometheus?

Reply:
moved to a separate commit